tests/testthat/_snaps/build.md

```diff
@@ -0,0 +1,113 @@
+      Updating deps/font-awesome-6.5.2/css/v4-shims.min.css
+      Updating deps/font-awesome-6.5.2/webfonts/fa-brands-400.ttf
+      Updating deps/font-awesome-6.5.2/webfonts/fa-brands-400.woff2
+      Updating deps/font-awesome-6.5.2/webfonts/fa-regular-400.ttf
+      Updating deps/font-awesome-6.5.2/webfonts/fa-regular-400.woff2
+      Updating deps/font-awesome-6.5.2/webfonts/fa-solid-900.ttf
+      Updating deps/font-awesome-6.5.2/webfonts/fa-solid-900.woff2
+      Updating deps/font-awesome-6.5.2/webfonts/fa-v4compatibility.ttf
+      Updating deps/font-awesome-6.5.2/webfonts/fa-v4compatibility.woff2
+      Updating deps/headroom-0.11.0/headroom.min.js
+      Updating deps/headroom-0.11.0/jQuery.headroom.min.js
+      Updating deps/jquery-3.6.0/jquery-3.6.0.js
+      Updating deps/jquery-3.6.0/jquery-3.6.0.min.js
+      Updating deps/jquery-3.6.0/jquery-3.6.0.min.map
+      Updating deps/search-1.0.0/autocomplete.jquery.min.js
+      Updating deps/search-1.0.0/fuse.min.js
+      Updating deps/search-1.0.0/mark.min.js
+      ── Building home ───────────────────────────────────────────────────────────────
+      Writing `authors.html`
+      Reading README.md
+      Writing `index.html`
+      Writing `index.html`
+      Writing `404.html`
+      ── Building function reference ─────────────────────────────────────────────────
+      Writing `reference/index.html`
+      Copying man/figures/kitten.jpg to reference/figures/kitten.jpg
+      Reading man/kitten.Rd
+      Writing `reference/kitten.html`
+      ── Building articles ───────────────────────────────────────────────────────────
+      Writing `articles/index.html`
+      Reading vignettes/kitten.Rmd
+      
+      
+      processing file: kitten.Rmd
+      1/10                  
+      2/10 [unnamed-chunk-1]
+      3/10                  
+      4/10 [unnamed-chunk-2]
+      5/10                  
+      6/10 [unnamed-chunk-3]
+      7/10                  
+      8/10 [magick]         
+      9/10                  
+      10/10 [plot]           
+      output file: /private/var/folders/7m/55bk34v53x12cjztb1fwxgvw0000gn/T/Rtmp7sGn6Y/kitten.knit.md
+      
+      /opt/homebrew/bin/pandoc +RTS -K512m -RTS /private/var/folders/7m/55bk34v53x12cjztb1fwxgvw0000gn/T/Rtmp7sGn6Y/kitten.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output <dst_path>/articles/kitten.html --lua-filter /Users/jayhesselberth/Library/R/arm64/4.5/library/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /Users/jayhesselberth/Library/R/arm64/4.5/library/rmarkdown/rmarkdown/lua/latex-div.lua --lua-filter /Users/jayhesselberth/Library/R/arm64/4.5/library/rmarkdown/rmarkdown/lua/table-classes.lua --standalone --section-divs --template /var/folders/7m/55bk34v53x12cjztb1fwxgvw0000gn/T//Rtmp7sGn6Y/pkgdown-rmd-template-11ef728bef6ba.html --highlight-style pygments --mathml --include-in-header /var/folders/7m/55bk34v53x12cjztb1fwxgvw0000gn/T//RtmpPvBjXL/rmarkdown-str11fc0c439f75.html 
```

Comment:
Many paths differe across GHA OS and the transform approach will be fiddly / brittle.